README.md

```diff
@@ -1,13 +1,11 @@
 <!-- markdownlint-disable no-inline-html -->
 
-<h1 align="center">
+<div align="center">
```

Comment:
This will break the styling on the VS Code marketplace a little bit, especially regarding the horizontal line. I want to see some space between the horizontal line and the text:

|  Old version | Your version  |
|---|---|
|  ![image](https://github.com/user-attachments/assets/da5b3cda-3493-42f8-b60b-e58dd6591316) |  ![image](https://github.com/user-attachments/assets/bb14d5e9-50e8-46dc-87c4-09f488bee849) |

Reply:
Okay

Reply:
Fixed